plugins/modules/junos_facts.py

```diff
@@ -18,8 +18,6 @@
   default, the module will collect basic fact information from the device to be included
   with the hostvars. Additional fact information can be collected based on the configured
   set of arguments.
-version_added: 1.0.0
-extends_documentation_fragment:
 - junipernetworks.junos.junos
```

Comment:
Looks like you removed slightly the wrong line here
```suggestion
version_added: 1.0.0
```